### PR TITLE
FAN-4088 | Add missing params for the F&P

### PR DESCRIPTION
**Pull request by vforge**

### CODE REVIEW

File: extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js [hunk 1]
```diff
@@ -5,10 +5,16 @@ require(['jquery'], function ($) {
 
 	$('#WikiaRail').one('afterLoad.rail', function() {
 		$.getScript(fpLibrary, function () {
+			var wikiName = $('meta[property="og:site_name"]').prop('content');
```
Comment (mklucsarits) [inline extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js:8]: Very creative 🥇

File: extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js [hunk 2]
```diff
@@ -5,10 +5,16 @@ require(['jquery'], function ($) {
...
+
 			// Load FP into a newly inserted element
 			$('#wikia-recent-activity').after('<div class="rail-module feed-posts-module"></div>');
 			var fpContainer = $('.feed-posts-module').get(0);
-			window.fp.default(fpContainer);
+			window.fp.default(fpContainer, {
+				communityName: wikiName,
+				communityId: wgCityId,
+				dbName: wgDBname,
```
Comment (mklucsarits) [inline extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js]: I think this dangling comma could be a problem for vanilla ES5.

Comment (vforge) [reply extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js]: Dang, true

Comment (vforge) [reply extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js]: Fixed!

**Comment by mklucsarits** [comment]
"Add missing comma"
*deletes comma*
:D

File: extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js (hunk 1 above)
Comment (KockaAdmiralac) [reply extensions/wikia/FeedsAndPosts/js/feedsAndPosts.js:8]: Wouldn't `mw.config.get('wgSiteName')` (or just `wgSiteName`, but that's deprecated in newer MediaWiki versions) be more consistent, though? Not like it matters much now that the pull request has been merged, but I feel like creativity isn't the point here.

**Pull request merged**